lib/vanagon/platform.rb

```diff
@@ -271,5 +272,19 @@ def is_cross_compiled?
     def is_cross_compiled_linux?
       return (is_cross_compiled? && is_linux?)
     end
+
+    # Utility matcher to determine if the platform uses host OS provided build tools
+    # (toolchain, boost, etc).
+    #
+    # @return [true, false] whether platform uses host OS toolchain, determined by
+    # whether the platform config set the use_os_toolchain flag (defaults to true
+    # if not set)
+    def uses_os_toolchain?
+      # Ensure we're returning a boolean:
+      return true if @use_os_toolchain == true || @use_os_toolchain == "true"
+      return false if @use_os_toolchain == false || @use_os_toolchain == "false"
```

Comment:
Should this also handle the `:true` and `:false` symbols? What about `1` and `0`? nil?